docs/source/installation.rst

```diff
@@ -19,40 +19,78 @@ distribution to view Conda's installation instructions.
 Install and Activate the environment
 ************************************
 
-Once `conda` is installed and available on the path, create and activate the environment using:
+For most end users, simply installing the package is sufficient to use ASPIRE.
+The commands in this section should install ASPIRE directly from the ``Python Package Index`` into your activated environment.
+This does not require checking out source code.
+If you are interested in checking out and working with the source code, running tests, or a different flavor of install,
+then skip to the next section now instead.
+
+Once ``conda`` is installed and available on the path, create and activate the environment using:
```

Comment:
For better understanding, suggest to change "create and activate the environment using:" to " create and activate aspire Python environment, as well as install the aspire package under the created virtual environment using:".  I feel there might be confusing if we don't say the difference between `aspire` and `aspire_env` for junior developers.

Reply:
We can do something like this.  `aspire` , `aspire_env` etc are just names, and they have be anything really... Personally, I just make a new environment for each issues.... I can see how this is confusing. I'll think about a change here. Thanks.

Reply:
I made a change here, should push in few minutes.